.github/actions/spelling/expect.txt

```diff
@@ -264,3 +264,5 @@ xts
 XUtn
 yournamespace
 YWQ
+backchannel
+OIDC
```

Comment:
I put this in the company dictionary so it can get removed from here

Reply:
Thanks Tim, done!